docs/manual/interpreters.md

```diff
@@ -60,3 +60,16 @@ Each Interpreter Setting can choose one of 'shared', 'scoped', 'isolated' interp
 In 'shared' mode, every notebook bound to the Interpreter Setting will share the single Interpreter instance. In 'scoped' mode, each notebook will create new Interpreter instance in the same interpreter process. In 'isolated' mode, each notebook will create new Interpreter process.
 
 <img src="../assets/themes/zeppelin/img/screenshots/interpreter_persession.png" width="400px">
+
+
+## Connecting to the Existing Remote Interpreter
+
+Zeppelin users can start interpreter thread embedded in their service. This will provide flexibility to user to start interpreter on remote host. To start interpreter along with your service you have to create an instance of ``RemoteInterpreterServer`` and start it as follows:
+
+``RemoteInterpreterServer interpreter=new RemoteInterpreterServer(3789); ``  
```

Comment:
@AhyoungRyu Thanks for review,Yes I think that will look better.Uploaded the changes please have a look